script/docker_entrypoint.clj

```diff
@@ -48,6 +48,7 @@
     (println "Launching cljdoc server")
     (process/exec "clojure"
                   "-J-Dcljdoc.host=0.0.0.0"
+                  "-J-XX:+CrashOnOutOfMemoryError"
```

Comment:
Will this result in the container restarting? Will we still be able to get the heap dump if that's the case? And if that's the case and it keeps crashing, will the heap dumps fill up the disk?

Reply:
Heap dumps should still be saved. I’ve set things up to save and compress the last 2 dumps.

Reply:
As for the heap dumps, I think @lread configured this in a way that only the last two heap dumps are kept.